.gitignore

```diff
@@ -2,3 +2,7 @@ Cargo.lock
 .*.swp
 /target
 tags
+/coresimd/target
+/stdsimd-test/target
+/stdsimd-test/assert-instr-macro/target
+/stdsimd-test/simd-test-macro/target
```

Comment:
is there a convention to ignore all paths containing target? `*/target/*` ?

Reply:
Oh you can just remove the leading `/` on the `/target` line near the top of the file